src/allocators.zig

```diff
@@ -469,6 +469,11 @@ pub fn BSSMap(comptime ValueType: type, comptime count: anytype, comptime store_
         const Self = @This();
         const Overflow = OverflowList(ValueType, count / 4);
 
+        const TRAILING_SLASH_CHAR = if (Environment.isWindows)
```

Comment:
instead of this, can't `std.fs.path.sep_str` can be used?

(additionally, this variable should be lowercase, but for this situation it should just be removed)

Reply:
you are right - replaced it